### PR TITLE
Login form working, moved some constants around.

### PR DESCRIPTION
**Pull request by mickel1138**

### CODE REVIEW

File: src/Entity/Account.php [hunk 1]
```diff
@@ -549,17 +547,18 @@ public function getRoles()
     {
         $roles = array(self::ROLE_USER);
         foreach ($this->getPerson()->getPersonDataRepositories() as $personDataRepository) {
-            if ($personDataRepository->getRole()->getName() == DataRepositoryRoles::MANAGER
+            dump('here');
```
Comment (praneethpr) [inline src/Entity/Account.php]: Remove dump

File: src/Security/LoginFormAuthenticator.php [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+     *
+     * @throws AuthenticationException When login is invalid.
+     *
+     * @return UserInterface Return the user.
+     */
+    public function getUser($credentials, UserProviderInterface $userProvider)
+    {
+        $username = $credentials['_username'];
+        
+        dump(Account::class);
```
Comment (praneethpr) [inline src/Security/LoginFormAuthenticator.php]: Remove dump statement

File: config/services.yaml [hunk 3]
```diff
@@ -28,4 +28,8 @@ services:
 
     App\Util\MaintenanceMode:
         bind:
-            string $maintenanceModeFilename: '%kernel.project_dir%%env(MAINTENANCE_MODE_FILENAME)%'
\ No newline at end of file
+            string $maintenanceModeFilename: '%kernel.project_dir%%env(MAINTENANCE_MODE_FILENAME)%'
+            
+    App\Security\LoginFormAuthenticator:
+        bind:
+            $maximumPasswordAge: "%env(ACCOUNT_PASSWORD_MAX_AGE)%"
```
Comment (mickel1138) [inline config/services.yaml]: new line

File: templates/Security/login.html.twig [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+                        <li>Abuse is subject to criminal prosecution.</li>
+                        <li>No expectation of privacy except as otherwise provided by applicable privacy laws.</li>
+                    </ul>
+                    <p>These warnings are required in accordance with Texas Administrative Code 202 as administered by the Texas Department of Information Resources.  Reference Chapter 33, Texas Penal Code.</p>
+                    <p></p>
+                </div>
+            </div>
+        </div> <!-- main table -->
+    </div>
+{% endblock %}
```
Comment (mickel1138) [inline templates/Security/login.html.twig]: new line

**Pull request merged**